charts/renku/templates/post-install-job.yaml

```diff
@@ -9,8 +9,7 @@ metadata:
   annotations:
     "helm.sh/hook": post-install,post-upgrade
     "helm.sh/hook-weight": "5"
-    "helm.sh/hook-delete-policy": "hook-succeeded,hook-failed"
-    # FIXME with newer version of helm "before-hook-creation"
+    "helm.sh/hook-delete-policy": "hook-succeeded,before-hook-creation"
```

Comment:
I would personally keep the job there after success too and go with a single value `"before-hook-creation"`.

---
https://github.com/kubernetes/helm/blob/master/docs/charts_hooks.md#automatically-delete-hook-from-previous-release